simplecpp.cpp

```diff
@@ -45,6 +45,11 @@ static bool isHex(const std::string &s)
     return s.size()>2 && (s.compare(0,2,"0x")==0 || s.compare(0,2,"0X")==0);
 }
 
+static bool isOct(const std::string &s)
+{
+    return s.size()>1 && (s.compare(0,1,"0")==0);
```

Comment:
please make this function more strict.. for instance, it should return false for `0xff`, `0.5`.. etc

Reply:
If you check that second char is a digit that is fine for me.

Reply:
Hi Daniel, you're right. I'll add test and check for second char. Stay tuned.

Reply:
Is this will be Ok to do as additional commit? Or should it be better if I create another branch with all of this in one commit and when do another pull request?

Reply:
please commit the fixes in your `octal-numbers` branch. When you push it, then it should be shown here in this Pull Request. You may need to use `--set-upstream` but git will tell you about it then.

Reply:
Already done.

Reply:
I've also checked new constFold() tests without isOct() and they fault. So it seems that isOct() works as expected.